File: sop-java/src/main/java/sop/enums/ArmorLabel.java
// SPDX-FileCopyrightText: 2021 Paul Schaub <vanitasvitae@fsfe.org>
//
// SPDX-License-Identifier: Apache-2.0

package sop.enums;

public enum ArmorLabel {
    Auto,
    Sig,
    Key,
    Cert,
    Message,
    ;

    @Override
    public String toString() {
        return super.toString().toLowerCase();
    }
}

File: sop-java/src/main/kotlin/sop/enums/ArmorLabel.kt
// SPDX-FileCopyrightText: 2023 Paul Schaub <vanitasvitae@fsfe.org>
//
// SPDX-License-Identifier: Apache-2.0

package sop.enums

@Deprecated("Use of armor labels is deprecated.")
enum class ArmorLabel {
    auto,
    sig,
    key,
    cert,
    message
}
